web/app.py

```diff
@@ -6,11 +6,29 @@
 pint = UnitRegistry()
 
 
+def normalize_unit(language_code, unit):
+    if language_code[:2].lower() in {"cs", "sk"}:
```

Comment:
Is trimming the language-code to two characters and lowercasing it necessary here?  We should be able to confirm whether all of the input datapaths already conform to two-character lowercase format.

Reply:
Hmm.. OK.  At the moment this _is_ necessary; it can in fact be a locale code.

Reply:
The requirements spec in #2 specifically states that we're intending to implement language-awareness, not locale-awareness, to begin with.  So I think we should adjust the [`crawler` service code](https://github.com/openculinary/crawler.git/) to extract only the language part from any potential locale codes.

Reply:
> The requirements spec in #2 specifically states that we're intending to implement language-awareness, not locale-awareness, to begin with. So I think we should adjust the [`crawler` service code](https://github.com/openculinary/crawler.git/) to extract only the language part from any potential locale codes.

Maybe: except then we'd reduce our ability to detect problems in upstream locale information.  Perhaps we should try to handle language codes that include locale information from the start.